File: epargne/ExternalApi/DTO/TransactionCourantDto.cs
using System;
using System.Text.Json.Serialization;
public class TransactionCourantDto
{
    [JsonPropertyName("idTransaction")]
    public int IdTransaction { get; set; }

    [JsonPropertyName("idCompte")]
    public int IdCompte { get; set; }

    [JsonPropertyName("dateTransaction")]
    public DateOnly DateTransaction { get; set; }

    [JsonPropertyName("libelle")]
    public string Libelle { get; set; }

    [JsonPropertyName("montant")]
    public decimal Montant { get; set; }

    [JsonPropertyName("sens")]
    public string Sens { get; set; }
}

File: server-ejb/src/main/java/com/example/dto/TransactionCourantDto.java
package com.example.dto;

import java.io.Serializable;
import java.time.LocalDate;

import lombok.Data;

@Data
public class TransactionCourantDto  implements Serializable{

    private Integer idTransaction;
    private Integer idCompte;        
    private LocalDate dateTransaction = LocalDate.now();
    private String libelle;
    private Double montant;
    private String sens;
    private boolean validate;
    private String devise;
}
